config/initializers/01_tag_presentation.rb

```diff
@@ -1,10 +1,4 @@
 module TagPresentation
-  module PresentationNameMethod
```

Comment:
Are you still able to run the generator for the openapi.json?  I think this is used there.

Reply:
@bdunne we don't generate the openapi.json we built that manually

Reply:
Oh, yikes.